client/tailscale/localclient.go

```diff
@@ -259,6 +259,23 @@ func (lc *LocalClient) DaemonMetrics(ctx context.Context) ([]byte, error) {
 	return lc.get200(ctx, "/localapi/v0/metrics")
 }
 
+// IncrementMetric increments the value of a Tailscale daemon's metric by
+// the given delta. If the metric has yet to exist, a new counter metric is
+// created and initialized to delta.
+func (lc *LocalClient) IncrementMetric(ctx context.Context, name string, delta int) error {
+	type metricUpdate struct {
+		Name  string `json:"name"`
+		Type  string `json:"type"`
+		Value int    `json:"value"` // amount to increment by
+	}
+	_, err := lc.send(ctx, "POST", "/localapi/v0/upload-client-metrics", 200, jsonBody([]metricUpdate{{
```

Comment:
return an error if delta is negative? Since we document this is only for counters.

(If we need gauges later, we can add a new method)

Reply:
I suppose could also rename this `IncrementCounter` if preferred, to leave open the obvious name for `IncrementGauge` later.